core/src/raw/tokio_util.rs

```diff
@@ -15,9 +15,10 @@
 // specific language governing permissions and limitations
 // under the License.
 
-use crate::*;
+use crate::{Error, ErrorKind};
 
 /// Parse tokio error into opendal::Error.
-pub fn new_task_join_error(e: tokio::task::JoinError) -> Error {
+// HACK - tokio::task::JoinError is not always present, use an existential type.
+pub fn new_task_join_error(e: impl Into<anyhow::Error>) -> Error {
```

Comment:
I'm thinking of adding a new feature called `internal-tokio-rt`. We can check if this feature has been enable here, and `layers-blocking` can include this feature instead of `tokio/rt`.

Reply:
The issue here is that even `layers-blocking` is disabled, this method (`new_task_join_error`) can be still referred. Where a new `internal-tokio-rt` doesn't help because the `tokio/rt` is enabled by other services feature.

Reply:
> layers-blocking can include this feature instead of tokio/rt

This can be done, while I don't find it makes difference.

Reply:
All services relying on `tokio::task::JoinError` must enable the `internal-tokio-rt` feature instead. This feature is added for remove `impl Into<anyhow::Error>` here, not for removing `tokio/rt` from more place.

Reply:
Updated.